src/data/boot.lua

```diff
@@ -187,6 +187,9 @@ function lovr.step()
   end
   local dt = lovr.timer.step()
   if lovr.audio then
+    if lovr.headset and lovr.headset.isPresent() then
+      lovr.headset.update(dt)
+    end
```

Comment:
I was wondering why mouselook wasn't working on the no game screen, and realized that it's because this is inside the `if lovr.audio` condition, and the nogame screen has the audio module disabled!